docs/migration.md

```diff
@@ -59,6 +59,13 @@ consider additional positioning prop support on a case-by-case basis.
 
 - `TooltipModal`: removed `popperModifiers` prop (see [note](#breaking-changes))
 
+#### @zendeskgarden/react-pagination
+
+- `Pagination`: renamed to `OffsetPagination`
+  - changed type export to `HTMLAttributes<HTMLElement>`
```

Comment:
```suggestion
  - changed type export from `HTMLAttributes<HTMLUListElement>` to `HTMLAttributes<HTMLElement>`
```

this minor change let's users know if they really need to care or not about the change

Reply:
Aha, thanks for the clarification. Added. :)